### PR TITLE
Fixed issue #2464 Error and success message appear after reset passwo…

### PR DESCRIPTION
**Pull request by aneteupeniece**
Issue was that an error was showing up on 2nd password reset attempt even if attempt was successful. Fixed issue

### CODE REVIEW

File: packages/scandipwa/src/route/PasswordChangePage/PasswordChangePage.container.js [hunk 1]
```diff
@@ -74,12 +74,12 @@ export class PasswordChangePageContainer extends PureComponent {
         isLoading: false
     };
 
-    static getDerivedStateFromProps(props) {
+    static getDerivedStateFromProps(props, state) {
         const { passwordResetStatus, showNotification } = props;
         const stateToBeUpdated = {};
+        stateToBeUpdated.isLoading = false;
```
Comment (yeegor) [inline packages/scandipwa/src/route/PasswordChangePage/PasswordChangePage.container.js:80]: why not do `const stateToBeUpdated = { isLoading: false };`

**Comment by carinadues** [comment]
@aneteupeniece could you fix this PR, please? We would like to use it for scandiPWA :)

**Comment by Elena-Evsa** [comment]
Only success message appears if attempt was successful

If user make mistake while entering password on second attempt and more, error message does not appear in https://pr-2558.d25kd40decjpue.amplifyapp.com/. 
Error message appears on PMREV on second and next try.

**Steps to reproduce:** 
1. Open FE
2. Forgot password
3. Enter registered email
4. Open email
5. Open link
6. Copy part of URL with token to PR link
7. Enter password using 1 class of characters (for example, 12345678)
8. Click Update password
9. Enter one more letter to each password using 2 classes of characters (for example, 12345678a)
10. Click Update password
11. Observe that error message does not appear if you will try to repeat steps 9-10 several times

**Comment by carinadues** [comment]
We're closing this PR since there were no updates.

**Pull request closed without merging**